### PR TITLE
Goutte's review 00

### PR DESCRIPTION
**Pull request by Goutte**
Miscellaneous active reading.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,35 +32,38 @@ This section guides you through submitting a bug report. Following these guideli
...
 Unsure where to begin contributing ? You can start by looking through these [Help wanted issues][help-wanted] issues.
 
-### Local development
+
+### Local Development
 
 #### Environment Variables
 
-You can edit .env to your liking (optional). This file should be .gitignored some time. 
+You can create and edit `.env.local` to your liking (optional).  This file will be ignored by git.
```
Comment (lutangar) [inline CONTRIBUTING.md:61]: It means it isn't?
Shall we ignore it right now?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,7 +77,7 @@ $ docker-compose up
 
 #### Aliases
 
-Some aliases are conveniently made available... 
+Some aliases are conveniently made available…
```
Comment (lutangar) [inline CONTRIBUTING.md:80]: I think you also used the makefile whereas it wasn't mentioned anywhere.
Could you add a note on it somewhere?

**Review by lutangar — APPROVED** [review]
I think you also mentioned some accessibility guidelines. Could you link me to these please?

**Review by JalilArfaoui — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Goutte) [reply CONTRIBUTING.md:80]: I did not use the makefile, since I threw the towel and went for local docker dev.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lutangar) [reply CONTRIBUTING.md:61]: Would you mind ignoring it in this PR then *bonvolu*? :blush:

File: CONTRIBUTING.md (hunk 2 above)
Comment (lutangar) [reply CONTRIBUTING.md:80]: Alright thx for the feedback.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Goutte) [reply CONTRIBUTING.md:61]: Well, the directives are already in `.gitignore`, I guess from a recipe of `symfony/framework-bundle`.

Comment (lutangar) [reply CONTRIBUTING.md:61]: Alright!

**Pull request merged**